users-api/routes/login.js

```diff
@@ -1,23 +1,24 @@
 //login.js
+const router = require('express').Router();
+const getDb = require('../db');
 
-const express = require('express');
-
-const router = express.Router();
-
-router.put('/', (req,res) => {
-  if (req.body.password == '1234'){
-    res.status(200).json
-    ({
-        "id":"abcd-efgh-ijkl-mnop",
-        "email":"test@test.com"
-    });
-  }
-  else{
-    res.status(403).json
-    ({
-        "message": "Invalid Credentials"
+router.post('/', (req,res) => {
+  getDb().then(db => {
+    db.collection('users').findOne(
+      {username: req.body.username}, (err, user) => {
+        if(err) {
+          console.log('ERROR.')
+          res.json(err)
```

Comment:
it should be more like `return res.status(500).json(err)` to have a bad status code, and prevent the rest of the function from running

Reply:
fixed.